packages/app/app/components/LyricsView/index.js

```diff
@@ -44,6 +70,63 @@ export class LyricsView extends React.Component {
       </div>
     );
   }
+  
+  //Addition of WildLeons
```

Comment:
This function should be kept somewhere separate from the component and supplied as a prop. Ideally the component needs just an event handler. All this complex logic should be isolated (maybe in the core package), we might want to use it somewhere else in the future (e.g. on mobile).